src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
On linux, they might be a gcc specific feature.  On Windows however, it looks like command files can be used for pretty much any command. https://support.microsoft.com/en-us/kb/830473

If we case this patch on windows, we shouldn't need to do any special casing in ```spawn()``` to dissect the command.

Sorry for the back and forth, but if we copy-and-paste 100 lines of distuitils into our code base, I just want to be sure that all other options have been explored.